docs/admin/performance_tuning.md

```diff
@@ -153,6 +153,74 @@ i2048 0      2Kb iovec Input only
     Number of repetitions, max inflight rpcs, message sizes can be adjusted based
     on the particular test/experiment.
 
+### Example: Without DAOS Agent
+
+For running `self_test` without a DAOS agent, the following environment variables needs to be
+defined:
+
+- `D_PROVIDER` defines mercury NA plugin and transport to be used (e.g.`ofi+verbs;ofi_rxm`)
+- `D_INTERFACE` defines network device name to be used (e.g. `ib0`)
+- `D_DOMAIN` defines the network name to be used (e.g. `mlx5_0:1`)
```

Comment:
i believe nowadays in almost all cases D_DOMAIN is optional and providing provider/interface is sufficient now for mercury to figure out proper domain on its own

Reply:
- [ ] Remove useless `D_DOMAIN` environment variable

Reply:
> * [x]  Remove useless `D_DOMAIN` environment variable

Fixed with commit 5b9f9649f.
`D_INTERFACE` and `D_DOMAIN` are working as expected.